modules/apps/transfer/types/token.go

```diff
@@ -12,6 +12,13 @@ import (
 // maxUint256 is the maximum value for a 256 bit unsigned integer.
 var maxUint256 = new(big.Int).Sub(new(big.Int).Lsh(big.NewInt(1), 256), big.NewInt(1))
 
+func NewToken(fullPath, amount string) Token {
```

Comment:
I'd rather not add another function to the API here, since we only use it a single place that is also a test. For that single test, you can just construct the Token manually with the same code as here.